src/platform/Zephyr/BLEAdvertisingArbiter.cpp

```diff
@@ -32,6 +34,11 @@ sys_slist_t sRequests;
 bool sIsInitialized = false;
 uint8_t sBtId       = 0;
 
+// TODO: Remove this if when all vendors have updated to Zephyr 3.6 or later.
+#if CHIP_DEVICE_LAYER_TARGET_NRFCONNECT
```

Comment:
Would it be possible to check the Zephyr version here and avoid a dedicated platform define (avoiding to add platform specific code in a common Zephyr Matter port file)  ?